src/Microsoft.IdentityModel.JsonWebTokens/JsonWebToken.cs

```diff
@@ -1031,14 +1030,9 @@ public DateTime IssuedAt
         /// If the 'iss' claim is not found, an empty string is returned.
         /// </para>
         /// </remarks>
-        public override string Issuer
-        {
-            get
-            {
-                _iss ??= Payload.GetStringValue(JwtRegisteredClaimNames.Iss);
-                return _iss;
-            }
-        }
+        public override string Issuer => Encoding.UTF8.GetString(IssuerUtf8.ToArray());
```

Comment:
- Encoding.UTF8.GetString with ReadOnlySpan overload is only available on .NET Core 2.1+ (not Net Fx).
   - I assume ToArray allocates a new array.
- Since IssuerUtf8 can change, we can't "cache" the string representation, so have to convert it to string each time.